src/AlgoliaSearchBundle.php

```diff
@@ -2,8 +2,17 @@
 
 namespace Algolia\SearchBundle;
 
+use AlgoliaSearch\Version;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
+use Symfony\Component\HttpKernel\Kernel as SfKernel;
 
 class AlgoliaSearchBundle extends Bundle
 {
+    public function boot()
+    {
+        parent::boot();
+
+        Version::addSuffixUserAgentSegment('Symfony', SfKernel::VERSION);
+        Version::addSuffixUserAgentSegment('Symfony Search Bundle', '3.0.0-BETA');
```

Comment:
My suggestion would be to no longer lock the version in here, but rather use https://packagist.org/packages/ocramius/package-versions to read the actually installed version using composer. This will avoid manual updates of the version number when releasing (which will be forgotten sooner or later).

Reply:
Interesting package, I'll look into it. I'd definitely like to have something automatic for it. A contributor recenlty made a [similar attempt on Scout](https://github.com/laravel/scout/pull/237).
Although, I might release the final 3.0 before this is done.

Reply:
Yeah, I wouldn't re-implement this for a library but rather look for an existing package. The solution you linked also modifies files in the vendor folder, which is always a bad idea.